docker/druid/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG ARG_OPENJDK_IMAGE=openjdk:8-jre-alpine
+
+FROM $ARG_OPENJDK_IMAGE
+
+ARG ARG_DRUID_VERSION
+ARG ARG_ZOOKEEPER_VERSION
+
+ENV DRUID_VERSION=$ARG_DRUID_VERSION
+ENV ZOOKEEPER_VERSION=$ARG_ZOOKEEPER_VERSION
+ENV LANG C.UTF-8
+
+RUN apk add --no-cache bash perl python curl
```

Comment:
This should install `python3` now, although that does require testing everything to see if it still works.